src/main/kotlin/com/zepben/evolve/services/network/translator/NetworkCimToProto.kt

```diff
@@ -653,6 +678,34 @@ fun Analog.toPb(): PBAnalog = toPb(this, PBAnalog.newBuilder()).build()
 fun Control.toPb(): PBControl = toPb(this, PBControl.newBuilder()).build()
 fun Discrete.toPb(): PBDiscrete = toPb(this, PBDiscrete.newBuilder()).build()
 
+/************ IEC61970 Base Protection ************/
+
+fun toPb(cim: CurrentRelay, pb: PBCurrentRelay.Builder): PBCurrentRelay.Builder =
+    pb.apply {
+        currentLimit1 = cim.currentLimit1 ?: UNKNOWN_DOUBLE
+        cim.inverseTimeFlag?.let { set = it } ?: setNull(NullValue.NULL_VALUE)
```

Comment:
@kgreav Sorry for not picking up on this earlier: consider renaming `set` and `null` to improve readability and future-proof for multiple nullable booleans.

Reply:
e.g. `inverseTimeFlagSet` and `inverseTimeFlagNull`

Reply:
yep true - i didn't give it much thought. will update evolve-grpc.